src/frontend/src/tests/lib/utils/token-card.utils.spec.ts

```diff
@@ -30,7 +22,7 @@ describe('mapHeaderData', () => {
 			decimals: ICP_TOKEN.decimals,
 			icon: ICP_TOKEN.icon,
 			network: ICP_NETWORK,
-			oisyName: { oisyName: BTC_MAINNET_TOKEN.name },
+			oisyName: { oisyName: ICP_NETWORK.name },
```

Comment:
It's fine but maybe for a safer test we should pick another token. The ICP network and token have the same name.

Reply:
right! I will use SEPOLIA_ETH in another PR:

- PR #